docs/examples/embeddings/jinaai_embeddings.ipynb

```diff
@@ -0,0 +1,383 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    llm = LiteLLM("command-nightly")
</pre><p>You also need to add 'COHERE_API_KEY' as environment variable to access their LLM via LiteLLM. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/run-llama/llama_index/pull/8704/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>